code/game/objects/structures/tables_racks.dm

```diff
@@ -100,11 +148,44 @@
 
 /obj/structure/table/examine(mob/user)
 	. = ..()
+	if(is_flipped)
+		. += span_notice("It's been flipped on its side!")
 	. += deconstruction_hints(user)
 
 /obj/structure/table/proc/deconstruction_hints(mob/user)
 	return span_notice("The top is <b>screwed</b> on, but the main <b>bolts</b> are also visible.")
 
+/obj/structure/table/proc/on_exit(datum/source, atom/movable/leaving, direction)
+	SIGNAL_HANDLER
+	if(!is_flipped)
+		return
```

Comment:
```suggestion
	if(!is_flipped)
		return
		
	if(leaving.movement_type & PHASING)
		return

	if(leaving == src)
		return
```
Otherwise will result in immovable tables